### ISSUE TITLE
Notary Application- South-Western Institute For Astronomy Research

### ISSUE THREAD

**Issue by IPFS-grandhelmsman** [filecoin-project/notary-governance#17]
# Notary Application

### Note: Please only submit PRs for proposed edits to this form. To apply as a Notary, please file an [issue](https://github.com/filecoin-project/notary-governance/issues/new?assignees=&labels=request&template=notary-application.md&title=)**.**

To apply as a notary, please fill out the following form.

## Core Information

- Name: Bailey Li
- (Optional) Affiliated Organization: South-Western Institute For Astronomy Research，SWIFAR
- Website / Social Media:  http://www.swifar.ynu.edu.cn/
- On-chain Address(es) to be Notarized: f3smifn52za5cunezhcrpdrcwd25bfylvqn5h2co7zs727dmk2lrkvvyxnrduumgobl2lidlvt353w4hq4acrq
- Region of Operation: Greater China
- Use case(s) to be supported:  Scientific Data (Satellite Imagery, Geological Data, Computer Vision, Autonomous Driving)
- DataCap Requested: 10PiB

*Please respond to the questions below in pargraph form, replacing the text saying "Please answer here". Include as much detail as you can in your answer!*

### Time Commitment

Describe the nature and duration of your affiliation with the Filecoin project. Please include relevant Github handles, miner ids, significant projects or contributions (with links).

```
GitHub handles：IPFS-grandhelmsman
slack handles：grandhelmsman 
Participated in Trusted Setup Ceremony Phase 1 https://github.com/arielgabizon/perpetualpowersoftau/blob/master/README.md
Participate in the first phase of the testnet that started in December 2019, ID: t019125, and the final storage capacity exceeds 1PiB
PB-scale Miner Testing,E-mail submitted the relevant test plan,the email was sent to the Filecoin team, and Filecoin team members like @why @keren knew about it https://github.com/filecoin-project/devgrants/blob/master/rfps/rfp-pbscale-miner-testing.md
Selected as a member of Filecoin Security Research Fellowship by Filecoin team,@why is one of the organizers, he can prove it https://security.filecoin.io/
Participated in Trusted Setup Ceremony Phase 2 https://filecoin.io/blog/trusted-setup-complete/
Participate in HackFS, the project is called Filecoin123 https://hack.ethglobal.co/showcase/filecoin123-recBa27E4zfpHRRln
Participated in Space Race, and finally ranked 22nd in the world (name: Five Star-Helmsman&Heiben) Miner ID: t03176 https://spacerace.filecoin.io/
Participated in the slingshot competition, project name: 1. Starry sky in Yunnan 2. Filecoin-Crawl
Participated in Slingshot-Phase 1 Closing Ceremony https://www.youtube.com/watch?v=qheNJZ1DClM&list=PL_0VrY55uV19nu6orLLCaGCGocsjvpiGZ&index=1

```

### Stake Exposure

Please cite total token at stake (currently available, locked as collateral, vesting over time) and any substantiating evidence.

```
SR1 reward: 26,000FIL
230,000 FIL with the miner f03176 as collaterals by the date of November 23, 2020.
```

### In-protocol Reputation

Please describe (in detail) your activity and tenure as a member of the Filecoin community. Please note (with links where possible) any contributions made to implementations of Filecoin, the spec, documentation, or to substantially help the Filecoin ecosystem grow.

```
1.Participated in Filecoin Devgrants and submitted three application project applications:
FileWallet-Tools https://github.com/filecoin-project/devgrants/pull/107
Filecoin-file-storage-and-retrieval-utility https://github.com/filecoin-project/devgrants/pull/106
Filecoin123-Block-Explorer https://github.com/filecoin-project/devgrants/pull/105
2.PB-scale Miner Testing,E-mail submitted the relevant test plan,the email was sent to the Filecoin team, and Filecoin team members like @why @keren knew about it https://github.com/filecoin-project/devgrants/blob/master/rfps/rfp-pbscale-miner-testing.md
3.Participate in HackFS, the project is called Filecoin123 https://hack.ethglobal.co/showcase/filecoin123-recBa27E4zfpHRRln
```

### In-protocol Security

Please describe your contributions to the security of Filecoin and the duration over which you've made contributions. Please also include any links or references who might be able to substantiate your contributions (e.g. if you've filed several bugs, please cite who you've communicated with on the Filecoin side).

```
We have submitted a lot of bugs that can help Filecoin improve its security. Here are some bugs
filecoin-project/rust-fil-proofs:
a sanity check failed of window post haven't catch.https://github.com/filecoin-project/rust-fil-proofs/issues/1308
The ability to read files in parallel is required when window post occurshttps://github.com/filecoin-project/rust-fil-proofs/issues/1281
Suggest to add more sector information in case of fault.https://github.com/filecoin-project/rust-fil-proofs/issues/1261

filecoin-project/lotus:
it's very slow to called `lotus-miner info` when the sectors number is over 10000.https://github.com/filecoin-project/lotus/issues/3913
miner attempting to prove commit over 200 sectors in epochhttps://github.com/filecoin-project/lotus/issues/3638
Some new deals from storage markets have append to a old sector which one have a invalid deal.https://github.com/filecoin-project/lotus/issues/3594
It seems have a dead lock in messagepool when push message.https://github.com/filecoin-project/lotus/issues/3115
`lotus state wait-msg` can not wait the `mpool replace` messages.https://github.com/filecoin-project/lotus/issues/2943
invalid proof (compute error?): failed to fill whole bufferhttps://github.com/filecoin-fivestar/lotus/issues/71
How can i get a rpc error return when define a chan response in rpc call?https://github.com/filecoin-project/go-jsonrpc/issues/14
Suggest adding IP output of client to log when JWT Verification failed.https://github.com/filecoin-project/lotus/issues/1870
```

### External Reputation

Please describe the nature of your organization, including the country of registration, size of the organization, and time since inception.

```
Name: South-Western Institute For Astronomy Research，SWIFAR
Organizational nature: Scientific research institution
Country of Registration: China
Established: September 2017
Organization introduction: 
South-Western Institute For Astronomy Research at Yunnan University (SWIFAR-YNU) was founded in September 2017. It serves as a major part of the University’s special discipline zone for astronomy, set up according to the “Yunnan University Development Plan for World-Class University” and “Yunnan University Development Plan for World-Class Astronomy Discipline”.  
By bringing in internationally renowned scientists as well as excellent young researchers, SWIFAR-YNU is dedicated to advance fundamental research for astronomy. The Institute currently has eleven full-time faculty members, specialized in studies of the interstellar medium, stars and stellar clusters, the Milky Way and nearby galaxies, galaxies and galaxy clusters, the large scale structure of the universe and cosmology. The SWIFAR-YNU near-field cosmology research group is the leading scientific organizer and undertaker of the huge LAMOST Galactic Spectroscopic Surveys, whereas its deep-field cosmology research group spearheads the cosmological applications of the weak-lensing effects, and is actively engaged in a number of international collaborative projects such as the Euclid, KiDS, CFHT Stripe-82 and VOICE surveys. The group is also a key member of the first national cosmic microwave background project AliCPT.    
SWIFAR-YNU is devoted to the national astronomy research capability development. Observing facilities currently under construction or development include the Yunnan University 50cm Photometric Telescope Array, the 1.6m Multi-Channel Photometric Survey Telescope (Mephisto). In collaboration with the Astronomy Department of Yunnan University, the Institute plans to build a high-performance computation and astronomical data centre.  
```

Please share any relevant details to help substantiate information about your organization (website, named officers, links to social media profiles).

```
Official website：http://www.swifar.ynu.edu.cn/
named officers: Haifeng WANG
Postdoctoral Fellow of Yunnan University (2018-present), LAMOST Fellow of Chinese Academy of Sciences (2018-present), Ph.D in Astrophysics of National Astronomical Observatories, Chinese Academy of Sciences  (2003-2018).
Research Interests: Chemo-Dynamical Structures of the Milky Way, Time Domain Stellar Physics, Near-Field Cosmology, Astronomical big data reduction and analysis, and etc.
```

Please share any relevant external information regarding your organization (e.g. news articles, social media profiles, etc.)

```
The content of the link is all in Chinese, you can view it through the translation software
http://www.xinhuanet.com/tech/2020-09/01/c_1126436418.htm
http://scitech.people.com.cn/n/2014/1030/c1007-25936449.html
https://baike.baidu.com/item/%E4%BA%91%E5%8D%97%E5%A4%A7%E5%AD%A6%E4%B8%AD%E5%9B%BD%E8%A5%BF%E5%8D%97%E5%A4%A9%E6%96%87%E7%A0%94%E7%A9%B6%E6%89%80/22229348?fr=aladdin
```

### Use Case Diversity

(Optional) Any additional information you'd like to share about the use case(s) you plan to support?

```
I hope to help and support the development of Filecoin. Participating in this Beta is my contribution to defining Filecoin+ roles and additional processes.
After the Beta period is over, I will also consider allocating time for this activity. Maybe by then, I will have a better understanding of different cases and get more value from them. Currently, I am happy to support any use case.
```

# Allocation Plan

### Allocation Strategy

How do you plan on allocating the DataCap requested above? Please describe your allocation strategy with as much specificity as you can.

```
After two conference calls, I think Philipp’s plan can be used for reference
Define thresholds with different levels of trust.The higher the requirements, the stricter the review：
Several important factors:
Reputation of the applicant organization
Is there any contribution to the Filecoin ecosystem (not necessarily a factor, but it will be a bonus item)
Scale of allocation
Assigned rating
Safety Precautions
Customer transparency
The type of data applied for (the public data set will be given priority)
The specific distribution plan can be viewed in the Google shared table：https://docs.google.com/spreadsheets/d/1AsAyaW1mV-eKyaqLWWTIAsaWHf24YNCriinHc-naJrs/edit?usp=sharing
```

Are there any internal processes you plan on impelementing regarding the target, amount, or rate at which you'll allocate DataCap?

```
Out of the principle of openness and transparency, there will not be too many internal processes
```

How do you plan on securing the DataCap to ensure your organization (and its delegated members) are the ones allocating the DataCap?

```
Fix the key of the address assigned to DataCap in the hardware wallet.
```

### Client Due Diligence

How will you vet your Client to ensure they are spending that DataCap responsibly?

```
I will need their information as well as evidence of customer reputation, data, diversity and decentralization, allocation strategies and tracking records.
In order to obtain the customer’s reputation and the customer’s business license, a screenshot of the search results of the search results of the customer’s name in the local corporate information query website, company website/social website link is required.
Divided into individuals and organizations
personal account:
Can use third-party authentication information
Organize customers:
Review organization entity information (registration information, etc.)
Corporate identity
In addition, I will review the performance of customers from time to time and cancel the qualifications for bad assignments.
```

What questions will you ask to ensure the Client can properly handle the DataCap you intend to allocate to them?

- What is your name, your company name? Can you provide your/company's social media/website link?
- What type of data do you want to store and how much data?
- Which address do you need to be notarized?
- How many miners do you want to store data to? How much data is for each miner?
- Will you record the allocation decision publicly?
- Which method/tool will you use to motivate miners? Which aspect of the miner will you check?
- Have you assigned before? If so, what did you do when you were asked?

What processes will you employ to confirm that a Client is not improperly over-allocating DataCap to a single entity?

```
1. Regularly check and review the usage of DataCap. Set up customer tracking records to prevent customers from making incorrect allocations. If an incorrect allocation is made, it will be recorded to reduce the subsequent allocation quantity
2. Use incentive mechanisms: encourage customers to report unfair distribution behavior. Encourage everyone to monitor each other's unfair distribution behavior, and encourage the unfair distribution behavior to be reported to GitHub or slack.
```

### Bookkeeping Plan

Do you plan on keeping records of your allocation decisions? If so, with what level of specificity do you intend to respond to any audit requests?

```
I intend to keep records of allocation decisions and make these records public in the GitHub repo. I will respond to all review requests
```

Do you plan on conduct your allocation decisions in public (e.g. Github repo), private (e.g. over email, Telegram, etc), or both?

```
Yes, I will publish these records in the GitHub repo. For the sake of transparency and fairness, I do not intend to send them via email or telegram.
```

### Past allocation

Have you previously received DataCap to allocate before? If so, please link to any previous applications.

```
None
```

Cumulatively, how much DataCap have you previously successfully allocated?

```
None
```

Have there been (or are there still) any disputes raised against you from your previous DataCap allocations?

```
None
```

**Comment by jnthnvctr** [comment]
Hi @IPFS-grandhelmsman  - we've taken a first pass of scoring your application here: https://docs.google.com/spreadsheets/d/1w84M0tOztB6g1vAmuGL2PVfpgfH_6yH8jF9izTfQObQ/edit?usp=sharing

Please note that there are some items that were not scored, and require further detail. Please check Column H and respond!

Please note: https://github.com/filecoin-project/notary-governance/pull/36 may affect your scoring if accepted!

**Comment by IPFS-grandhelmsman** [comment]
I do not have permission to access the link, please grant me. This is my account number: 13169628872@163.com

**Comment by IPFS-grandhelmsman** [comment]
Answers to the questions in the form:
H11：https://docs.google.com/spreadsheets/d/1w84M0tOztB6g1vAmuGL2PVfpgfH_6yH8jF9izTfQObQ/edit#gid=0&range=H11
The main body of this application is SWIFAR, and Grandhelmsman is its subordinate cooperation team. They are focusing on different directions. Grandhelmsman focuses on the research and development of major public chains in the blockchain, and has focused on Filecoin project research in the past two years.
SWIFAR is the Institute of Astronomy. The Institute currently has 11 full-time researchers. The research fields involve interstellar media, stars and star clusters, the Milky Way and neighboring galaxies, galaxies and galaxy clusters, large-scale structures of the universe, and cosmology. And in the past few years, a large amount of astronomical data has been accumulated. In the process of working with the Grandhelmsman team, I learned about the Filecoin project, recognized its values, and firmly believed that storing valid data can change the human process. So they also store part of the data in the Filecoin network. Applying for Notary this time, I also hope to identify more valid data through their expertise in astronomical data so that it can be stored in the Filecoin network.

H21：https://docs.google.com/spreadsheets/d/1w84M0tOztB6g1vAmuGL2PVfpgfH_6yH8jF9izTfQObQ/edit#gid=0&range=H21
The corresponding threshold will be adjusted. Because I applied for the initial value of 10PiB, the threshold may seem to be very large in the initial form design. I have also seen that there are still many miners with very small amounts of data in the network, so this table should be adjusted at the beginning.
As the network grows day by day, the amount of stored data will definitely increase, so it is possible to follow the current table later.

H23：https://docs.google.com/spreadsheets/d/1w84M0tOztB6g1vAmuGL2PVfpgfH_6yH8jF9izTfQObQ/edit#gid=0&range=H23
Yes, I refer to the distribution tool of the Filecoin Foundation, and many functions are worth learning from. The tool is currently being designed and developed. After your reminder, I will use the ledger to sign and process transactions.

**Comment by IPFS-grandhelmsman** [comment]
Modified Client Due Diligence
How will you vet your Client to ensure they are spending that DataCap responsibly?

I will need their information as well as evidence of customer reputation, data, diversity and decentralization, allocation strategies and tracking records.
In order to obtain the customer’s reputation and the customer’s business license, a screenshot of the search results of the search results of the customer’s name in the local corporate information query website, company website/social website link is required.
Divided into individuals and organizations
personal account:
Can use third-party authentication information
Organize customers:
Review organization entity information (registration information, etc.)
Corporate identity
In addition, I will review the performance of customers from time to time and cancel the qualifications for bad assignments.

What questions will you ask to ensure the Client can properly handle the DataCap you intend to allocate to them?

- What is your name, your company name? Can you provide your/company's social media/website link?
- What type of data do you want to store and how much data?
- Which address do you need to be notarized?
- How many miners do you want to store data to? How much data is for each miner?
- Will you record the allocation decision publicly?
- Which method/tool will you use to motivate miners? Which aspect of the miner will you check?
- Have you assigned before? If so, what did you do when you were asked?

What processes will you employ to confirm that a Client is not improperly over-allocating DataCap to a single entity?
1. Regularly check and review the usage of DataCap. Set up customer tracking records to prevent customers from making incorrect allocations. If an incorrect allocation is made, it will be recorded to reduce the subsequent allocation quantity
2. Use incentive mechanisms: encourage customers to report unfair distribution behavior. Encourage everyone to monitor each other's unfair distribution behavior, and encourage the unfair distribution behavior to be reported to GitHub or slack.

**Comment by jnthnvctr** [comment]
hi @IPFS-grandhelmsman  - we've scored your application here: https://docs.google.com/spreadsheets/d/10Y64rSgnsVqN_KsJAokrfqvXZYgIYfHpMiT6_HnlKKA/edit?usp=sharing

Eligibility score: 2
Unrounded score: 1.7

**Comment by dkkapur** [comment]
@IPFS-grandhelmsman  - just wanted to let you know that Notary applications are currently open again until May 25. In case you'd be interested in applying again, please create a new Issue of type "Notary Application" and fill in the form (it is completely reasonable to reuse material from your prior application).
